node/src/main/scala/org/bitcoins/node/NeutrinoNode.scala

```diff
@@ -78,13 +74,25 @@ case class NeutrinoNode(
       BlockHeaderDAO()(executionContext, chainConfig).getBlockchains()
     for {
       chainApi <- chainApiFromDb()
+      _ <- chainApi.getBestBlockHash()
+
+      syncPeer <- peerManager.randomPeerWithService(_.nodeCompactFilters)
+      _ = logger.info(s"Starting sync with $syncPeer")
```

Comment:
Lets change this log to `Syncing with $syncPeer`, its a little confusing to look at logs and see that we keep `Starting sync with peer` and I immediately thought, why does it keep restarting?! 

It turns out it is just querying data? Maybe thats better term? :man_shrugging: 

![Screenshot from 2022-06-24 07-47-06](https://user-images.githubusercontent.com/3514957/175539812-adb5e29c-de00-4bf6-b361-377fc41abed3.png)